corfudb-tools/src/main/java/org/corfudb/browser/CorfuBrowserMain.java

```diff
@@ -0,0 +1,48 @@
+package org.corfudb.browser;
+
+import java.util.Map;
+
+import org.corfudb.util.GitRepositoryState;
+import org.docopt.Docopt;
+
+public class CorfuBrowserMain {
+    private static final String USAGE = "Usage: corfu-browser --host=<host> " +
+        "--port=<port> --namespace=<namespace> --tablename=<tablename> " +
+        "--keystore=<keystore_file> --ks_password=<keystore_password> " +
```

Comment:
We can make these fields optional.
This will be helpful when we want to browse data on our local setup which might not have the tls requirements.
For optional parameters you can take a look at CorfuServer Docopt. (You need to enclose optional parameters in '[, ]')

Reply:
done